zebra/connected.c

```diff
@@ -393,10 +393,10 @@ void connected_down(struct interface *ifp, struct connected *ifc)
 	 * head.
 	 */
 	rib_delete(afi, SAFI_UNICAST, zvrf->vrf->vrf_id, ZEBRA_ROUTE_CONNECT, 0,
-		   0, &p, NULL, &nh, 0, zvrf->table_id, 0, 0, false);
+		   0, &p, NULL, &nh, 0, zvrf->table_id, 0, 0, false, true);
```

Comment:
Not a comment about this fix, but thats a lot of parameters being passed to the function. Its easy to get wrong at some point and mix up the trues with the falses, I fear. I also know that this is just old code that's been extended over time.